api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -250,7 +250,20 @@ message Bootstrap {
   // Setting this value causes failure if the
   // ``envoy.restart_features.use_apple_api_for_dns_lookups`` runtime value is true during
   // server startup. Apple' API only uses UDP for DNS resolution.
-  bool use_tcp_for_dns_lookups = 20;
+  //
+  // .. attention::
```

Comment:
We could probably do without this line.  The comment below will suffice.

Reply:
Removed them from all .proto files.